### PR TITLE
Update credits-faq.html

### PR DESCRIPTION
**Pull request by gurumukhi**
Removed the text mentioning we send mail if more info is needed for credits application.

@gerv please review.

### CODE REVIEW

**Comment by gurumukhi** [comment]
@gerv could you please also check the issue with auto-merge here?

**Comment by gurumukhi** [comment]
@gerv can you please check if this PR requires manual merging?

**Comment by alexgibson** [comment]
@gurumukhi hi, Gerv does not have commit rights to this repo. Merging requires review from a bedrock commiter.

File: bedrock/mozorg/templates/mozorg/credits-faq.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h2>Why do I need to include a citation?</h2>
 
 <h2>When will I get added, and how do I know?</h2>
 
-<p>Additions are normally done in batches, a few months apart. You will receive an email when your name is added, or requesting more information if there has been a problem with your application (e.g. additional verification is needed). When you have been added, you will see your name and citation in <a href="https://github.com/mozilla/community-data/commits/master/credits/names.csv">the log</a>.</p>
+<p>Additions are normally done in batches, a few months apart, you will receive an email when your name is added. When you have been added, you will see your name and citation in <a href="https://github.com/mozilla/community-data/commits/master/credits/names.csv">the log</a>. Your name will not be added if your application does not meet the criteria mentioned above, in that case feel free to re-apply once you have a stronger citation and evidences.</p>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/credits-faq.html]: Nit: I'm not sure the changes here read quite right as sentances in their own merit. It makes sense overall, but I'd suggest revising the punctuation slightly.

**Comment by alexgibson** [comment]
Friendly ping @gurumukhi - is this something you still have time to work on?

File: bedrock/mozorg/templates/mozorg/credits-faq.html (hunk 1 above)
Comment (gurumukhi) [reply bedrock/mozorg/templates/mozorg/credits-faq.html]: How about following text? If this looks good to you, I will update the PR. (also notice I updated word log to logs below)..

Additions are normally done in batches, usually a few months apart. You will receive an email when your name is added and you will also see your name and citation in <a href="https://github.com/mozilla/community-data/commits/master/credits/names.csv">the logs</a>. If your application does not meet the criteria mentioned above, your name will not be added and in that case re-apply once you have a stronger citation and evidences.

**Comment by gurumukhi** [comment]
Thanks for the reminder @alexgibson, I added another comment in the discussion (https://github.com/mozilla/bedrock/pull/5410#discussion_r167969772), once we finalize the text content, I will update the PR accordingly. Thanks :)

File: bedrock/mozorg/templates/mozorg/credits-faq.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/credits-faq.html]: First sentance reads nice. For the second, how about:

```
If your application does not meet the criteria mentioned above, your name will not be added. You are always welcome to re-apply once you have a stronger citation and further evidence.
```

**Comment by gurumukhi** [comment]
Updated the PR as per latest suggestions in text :+1:

**Pull request merged**